### ISSUE TITLE
Startup local rescanning is caused by “Rescan now” shying away in a corner

### ISSUE THREAD

**Issue by shenlebantongying** [xiaoyifang/goldendict-ng#1606]
tl;dr: changes that makes sence

* Option to disable startup rescanning
* Add a button ing Edit menu for easy rescan without restart or opening the dictionary dialog

Info
---
Usability issues:

* Startup checking is not really necessary https://github.com/xiaoyifang/goldendict-ng/issues/504
* The tab is very loooong and may grow in the future, new tabs are maybe invisible unless the user has a giant screen.
   * or turn it to [vertical](https://github.com/xiaoyifang/goldendict-ng/issues/397#issuecomment-1475147136) 
* Add a new dict to while adding files when a path is already in "Files", the user needs to open the dialog then close.
* Editing other “somewhat dictionaries” like “Programs” doesn't really need to "rescan" at all
* Cannot open Edit Groups dialog directly.
* There is no "rescan now" button in "Sound Dir" tab.
* Both The opening and "Ok" button of this dialog carries lots of responsibilities

<img width="400px" src="https://xiaoyifang.github.io/goldendict-ng/img/dict_file_tab.webp">

How about[^1]
* Split them into different dialogs
* Change the modality of "Inspect dictionaries" and "Manage groups" dialog, so that "Edit local files", "Inspect dictionaries" and "Manage groups" can be opened in parallel. (Maybe a bad idea.)
* Remove startup scan
* Automatic "Rescan now" in the “local dictionary” dialog only
* Add a quick "Rescan/Reindex now" in menu

Outcome

<img width="207px" src="https://github.com/xiaoyifang/goldendict-ng/assets/20123683/c7d69ac2-9f1e-4fee-88f1-91a0ca5d8e7c">

[^1]: I am pretty aware of the cost of implementing this, this is yet another idea. 😅

**Comment by xiaoyifang** [comment]
> * Startup checking is not really necessary [boost startup speed  #504](https://github.com/xiaoyifang/goldendict-ng/issues/504)

the startup checking based on the following assumption that 
the dictionaries have changed before the gd-ng start.

Without the startup checking ,  the gd-ng will lost the chance to reflect the new changes in the dictionary.

**Comment by xiaoyifang** [comment]
> * Change the modality of "Inspect dictionaries" and "Manage groups" dialog, so that "Edit local files", "Inspect dictionaries" and "Manage groups" can be opened in parallel. (Maybe a bad idea.)

groups depends on dictionaries .

**Comment by xiaoyifang** [comment]
> * Automatic "Rescan now" in the “local dictionary” dialog only
> * Add a quick "Rescan/Reindex now" in menu

Sound ok
